release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -912,6 +903,12 @@ With each major, minor, or patch version release of {product-title}, two compone
 
 The `HostedCluster` and `NodePool` API resources are available in the beta version of the API and follow a similar policy to xref:../rest_api/understanding-api-support-tiers.adoc[{product-title}] and link:https://kubernetes.io/docs/concepts/overview/kubernetes-api/[Kubernetes].
 
+[id="ocp-4-12-rhv"]
+=== Red Hat Virtualization (RHV)
```

Comment:
This subsection doesn't follow the format of the other sections in the New Features and Enhancements section where it goes:

```
| New features and Enhancements
|____ Component
|________ enhancement header
|____________ body text
```
Not sure if that's an issue, just noting it here 🙂

Reply:
I _think_ it is okay. The we have used some bullets in the admin/dev perspectives for the web console section of the release notes